helpers/installcredprovider.ps1

```diff
@@ -0,0 +1,61 @@
+# A PowerShell script that adds the latest version of the Azure Artifacts credential provider
+# plugin for Dotnet and/or NuGet to ~/.nuget/plugins directory
+# To install netfx and netcore, run installcredprovider.ps1
+# To install only netcore, run installcredprovider.ps1 -AddNetfx false
+# More: https://github.com/Microsoft/artifacts-credprovider/blob/master/README.md
+
+param(
+    $AddNetfx = $True
```

Comment:
Can we make this false by default?

Reply:
Yes, I'll also update the comment

Reply:
Change this to a switch instead (which is false by default) i.e. param([switch] $AddNetfx). That means to make netfx installable you just specify -AddNetfx